using ExceptionExe;
int[] myNumbers = { 10, 20, 30,40 };

exceptionExe obj1 = new exceptionExe();
obj1.Check(myNumbers);

exceptionExe obj2 = new exceptionExe();
obj2.CheckT(11);

exceptionExe obj3 = new exceptionExe();
obj3.MoValidation("+917899669078");








